integration-tests/src/chains/rococo/genesis.rs

```diff
@@ -3,7 +3,7 @@ use emulated_integration_tests_common::{
 	validators,
 };
 use polkadot_primitives::{AssignmentId, ValidatorId};
-use pop_runtime::Balance;
+use pop_runtime_common::Balance;
```

Comment:
This seems weired, Rococo genesis shouldnt need to know about pop. Seems it isnt being used in any case, so can probably be removed.